Add entry for AMS load_balancer.url config option (#159)
anbox-cloud/discourse-docs: ref/ams-configuration.md

| `images.update_interval` | string | `5m` | Frequency of image updates (for example: 1h, 30m). |
| `images.url` | string | `https://images.anbox-cloud.io/stable/` | URL of the image server to use. |
| `images.version_lockstep` | bool | true | Whether to put the version of the latest pulled image and the AMS version in a lockstep. This ensures that a deployment is not automatically updated to newer image versions if AMS is still at an older version. This only applies for new major and minor but not patch version updates. |

> | `load_balancer.url` | string | - | URL of the load balancer AMS sits behind. The URL will be handed to containers started by AMS to allow them contact AMS through the load blancer and not via the address of an individual AMS instance. |

Reviewer: ```suggestion
| `load_balancer.url` | string | - | URL of the load balancer behind which AMS sits. The URL is handed to containers started by AMS to allow them to contact AMS through the load balancer and not via the address of an individual AMS instance. |
```
